examples/java/README.md

```diff
@@ -20,16 +17,43 @@ To use a prebuilt library (the easy way), follow these steps:
     sudo make install
     ```
 
-2. On Eclipse, click "Import" -> "Existing Projects into Workspace" and select the root directory: `sdk/examples/java/JavaBindingSample`
-3. Open the "Properties" of the project and go to "Java Build Path" -> "Source"
-4. Click "Link Source..." and select the folder `sdk/bindings/java`
-5. Copy the library `libmegajava.so` into the `libs` folder in your project.
+2. Copy the library `libmegajava.so` into the `libs` folder in your project.
     
     ```
-    cp bindings/java/.lib/libmegajava.so examples/java/libs
+    cp bindings/java/.libs/libmegajava.so examples/java/JavaBindingSample/libs
     ```
     
-6. Build and run the project in Eclipse
+3. Copy the Java classes into the `src` folder in your project.
+
+    ```
+    cp bindings/java/nz/mega/sdk/*.java examples/java/JavaBindingSample/src/nz/mega/sdk
+    ```
+    
+4. In Eclipse, click "Import" -> "Existing Projects into Workspace" and select the root directory: `sdk/examples/java`
```

Comment:
If the Eclipse project files are relocated elsewhere (out of the way of the root, e. g into `contrib/Eclipse`), then there needs to be a word about this. In this case, it might make sense to create a section for Eclipse detailing the steps:
```How to import the project into Eclipse```